src/olympia/versions/migrations/0026_versionreviewerflags_pending_content_rejection.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.14 on 2022-07-20 13:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('versions', '0025_auto_20220614_1653'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='versionreviewerflags',
+            name='pending_content_rejection',
+            field=models.BooleanField(default=False),
```

Comment:
Needs to be a `NullBooleanField` for backwards-compatibility since Django won't generate a boolean `DEFAULT` in database (or rather it will but then drop it, so it's not 100% backwards-compatible)

If you want to avoid multiple migrations afterwards, don't modify the model, only change the migration to make it a `NullBooleanField`, keeping `default=False`.

Reply:
Not following here - we want the django behaviour don't we?  It'd set all the existing records to `False` by defining the default.  Are we just concerned about the incompatibility during the push window?  (if so, could just ask reviewers not to submit during that 30 minute window?)

Reply:
I'm worried about the incompatibility during the push window and lack of backwards-compatibility if we need to go back to previous tag/cancel the push midway.

Reply:
https://github.com/mozilla/addons-server/pull/19490/commits/0b4e06f7609601d195cd90f1653016d9ad94096b